pkg/kubelet/kubelet_test.go

```diff
@@ -3529,10 +3598,27 @@ func TestIsPodResizeInProgress(t *testing.T) {
 		name:         "resizing cpu limit",
 		status:       mkPodStatus(resizeCPULimitC1Status, steadyStateC2Status),
 		expectResize: true,
-	}}
+	},
+		{
+			name:         "resizing cpu request for restartable init container",
```

Comment:
nitpick: the indentation looks incorrect. You can run `./hack/update-gofmt.sh`.

Reply:
Done.